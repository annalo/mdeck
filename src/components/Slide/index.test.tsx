import React from "react";
import { render } from "@testing-library/react";

import { Slide } from ".";

const component = (
  <Slide htmlString="<h1>Title</h1>">
    <div>Some Child Element</div>
  </Slide>
);

describe("<Slide />", () => {
  test("should render and match the snapshot", () => {
    const { asFragment } = render(component);
    expect(asFragment()).toMatchSnapshot();
  });
});
