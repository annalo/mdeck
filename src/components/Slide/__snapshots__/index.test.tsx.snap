// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slide /> should render and match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Title
    </h1>
  </div>
</DocumentFragment>
`;
